### DISCUSSION TITLE
通过SparkDorisConnector写入数据时，如何删除数据？

### DISCUSSION THREAD

**Discussion by jwsmai** [apache/doris#13721] [A - General / Q&A]
表的模型是Unique，一般情况下会根据Unique Key对数据做Replace，但是有些错误的数据无法根据Unique Key删除。

**Comment by yuanyuan8983** [comment]
I don't know what you mean. The unique model supports delete. You can chat with me through email for specific use. Let's have a look. lyy8983@163.com
